libs/core/src/utils/form.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { isRequired } from './form';
+
+describe('isRequired', () => {
+  it('returns empty string when no target or component defined', () => {
+    expect(isRequired(undefined, undefined)).toEqual(undefined);
+  });
+});
```

Comment:
We should have multiple test for `assignDescription` that covers the various scenarios.

* helperMessage null or undefined
* invalid both true or false
* happy path scenarios

We also need to add test for `messageId`